### PR TITLE
NIAD-121 Shell scripts for making requests to MESH API

### PR DESCRIPTION
**Pull request by mattd-kainos**
Shell scripts for making requests to MESH API for verification and manual testing

### CODE REVIEW

File: mesh/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Authenticate
+
+MESH API docs says to do this first but our experience seems that its not needed.
+
+    ./mesh.sh auth
+    
+### List Inbox Messages
+
+List all messages in the inbox. The JSON response is printed to the console.
```
Comment (grzegorz-robak) [inline mesh/README.md]: It's first 500, not all messages. Also it lists only message ids.

**Review by grzegorz-robak — DISMISSED** [review]

File: mesh/GPHA_SAMPLE.dat [hunk 2]
```diff
@@ -0,0 +1,44 @@
+UNB+UNOA:2+TES5+XX11+920117:1021+00000453'
```
Comment (bartek-sarul) [inline mesh/GPHA_SAMPLE.dat:1]: Maybe rename file to `GPHA_SAMPLE.edifact` ?

**Review by bartek-sarul — CHANGES_REQUESTED** [review]

File: mesh/GPHA_SAMPLE.dat (hunk 2 above)
Comment (mattd-kainos) [reply mesh/GPHA_SAMPLE.dat:1]: Convention used by NHSD is naming these .dat

File: mesh/README.md (hunk 1 above)
Comment (mattd-kainos) [reply mesh/README.md]: fixed by just referring to mesh api docs

**Review by grzegorz-robak — DISMISSED** [review]

**Review by bartek-sarul — APPROVED** [review]

**Review by grzegorz-robak — APPROVED** [review]

**Pull request merged**